src/Mvc/Mvc.ViewFeatures/src/Rendering/IHtmlHelper.cs

```diff
@@ -375,7 +375,9 @@ public interface IHtmlHelper
     /// <typeparam name="TEnum">Type to generate a select list for.</typeparam>
     /// <returns>
     /// An <see cref="IEnumerable{SelectListItem}"/> containing the select list for the given
-    /// <typeparamref name="TEnum"/>.
+    /// <typeparamref name="TEnum"/>,
+    /// with a decimal representation of the ordinal as <see cref="SelectListItem.Value" />
+    /// and the display name as <see cref="SelectListItem#Text" >.
```

Comment:
It's likely going to complain about this next:

```suggestion
    /// and the display name as <see cref="SelectListItem.Text" >.
```